SECURITY.md

```diff
@@ -22,11 +22,42 @@ If it is not triaged in a couple of days, feel free to tag `@the-frey` or `@jake
 
 ## Critical bug or security issue 💥
 
-For a critical security bug, please do not publicly disclose it, or post on GitHub. Instead, go to [Discord](https://discord.gg/wHdzjS5vXx), and alert the core engineers:
+If you're here because you're trying to figure out how to notify us of a security issue, 
+
+go to [Discord](https://discord.gg/wHdzjS5vXx), and alert the core engineers:
+
 
 - Jake (Meow) `Meow Stargaze ✨🔭#1736`
 - Dimi `dimi 🦙#2998`
 - Alex (the-frey) `the-frey#8626`
 - Ben2x4 `Ben2x4#4071`
 - Jacob `jacobgadikian#9883`
 - Giansalex `giansalex#1825`
+
+
+Please avoid opening public issues on Github that contain information about a
+potential security vulnerability as this makes it difficult to reduce the impact
+and harm of valid security issues.
+
+### Coordinated Vulnerability Disclosure Policy
+
+We ask security researchers to keep vulnerabilities and communications around vulnerability submissions private and confidential until a patch is developed to protect the people using Tendermint’s protocols. In addition to this, we ask that you:
+
+- Allow us a reasonable amount of time to correct or address security vulnerabilities.
+- Avoid exploiting any vulnerabilities that you discover.
+- Demonstrate good faith by not disrupting or degrading Juno’s network, data, or services.
+
+### Vulnerability Disclosure Process
+
+Juno uses the following disclosure process:
+
+- Once a security report is received, the Juno core development team works to verify the issue and confirm its severity level using CVSS.
+- Patches are prepared for eligible releases in private repositories.
+- If it is determined that a CVE-ID is required, we request a CVE through a CVE Numbering Authority.
```

Comment:
Think this is very unlikely given how far we are up the stack. If we found something in Tendermint, we would disclose it them privately, they're the most likely to need this sort of thing...

Reply:
Because while technically correct, I mean, I hope we never need to request a CVE ID